src/CMakeLists.txt

```diff
@@ -268,13 +268,8 @@ list(APPEND SM_WINDOWS_PROGRAM_DLLS
   "${SM_PROGRAM_DIR}/avcodec-53.dll"
   "${SM_PROGRAM_DIR}/avformat-53.dll"
   "${SM_PROGRAM_DIR}/avutil-51.dll"
-  "${SM_PROGRAM_DIR}/msvcp100.dll"
```

Comment:
Windows builds use Zeranoe's ffmpeg shared libraries. They are built with MinGW and linked to msvcrt.dll. That's why.